puresasl/mechanisms.py

```diff
@@ -13,6 +14,11 @@
 except ImportError:
     have_kerberos = False
 
+if platform.system() == 'Windows' :
+    import winkerberos as kerberos
+    kerberos.authGSSClientUserName = kerberos.authGSSClientUsername
```

Comment:
Is this line performing some magic that I can't figure out, or is it just leftover?

Reply:
It is doing some magic, the winkerberos library has different capitalisation on the 'N' in name. this just makes it compatible.

Reply:
Ahhh, okay, that makes sense. Can you just add a comment about that? Other than that, I think this is probably good to go.